### PR TITLE
Make the terms [deleted] and [removed] more explicit.

### PR DESCRIPTION
**Pull request by mistyhands**
For some users, these terms are seemingly interchangeable.

### CODE REVIEW

**Comment by Pokechu22** [comment]
Just for reference, `git rebase -i master` would let you merge the changes together, and then they could be pushed with `git push --force` onto your local branch -- no need to start a new PR.  (And also, this was originally #1444 &mdash; there's some discussion there).

File: r2/r2/lib/jsontemplates.py [hunk 1]
```diff
@@ -690,7 +690,7 @@ def thing_attr(self, thing, attr):
             if not thing.expunged:
                 return safemarkdown(thing.selftext)
             else:
-                return safemarkdown(_("[removed]"))
+                return safemarkdown(_("[removed by moderator]"))
```
Comment (13steinj) [inline r2/r2/lib/jsontemplates.py]: You forgot about L688

**Comment by 13steinj** [comment]
@Pokechu22 Given the fact that he made three commits instead of one, I assume he's working in browser, not via git; there's no rebase tool via the browser.

File: r2/r2/lib/jsontemplates.py (hunk 1 above)
Comment (minecast) [inline r2/r2/lib/jsontemplates.py]: @13steinj I figured I'd just leave it blank as it was before.

**Comment by minecast** [comment]
@13steinj The other 2 commits were me being stupid, unrelated to this particular commit. But yeah, I'm a git noob.

**Comment by umbrae** [comment]
We want to do this, but we'd like to be a little bit more comprehensive about it, and particularly denote when something was removed by an admin. Do you want to try to tackle that also?

**Comment by minecast** [comment]
@umbrae I'd be glad to! However, I'm quite unfamiliar with the reddit source. Would item.moderator return the user who removed an item?

**Comment by 13steinj** [comment]
@minecast Not really in this case.

I'll send a PR to your branch when I can. It's determined via `.verdict` and the string that it is. And...There's A small bug in removals that I'm currently fixing regarding the order in which the text is determined, so it may be better to wait until that's reviewed, since otherwise if I send a patch to your branch first that bug would only get worse.

**Comment by minecast** [comment]
@13steinj Thank you!

**Comment by minecast** [comment]
@umbrae added admin distinction & rebased

**Comment by 13steinj** [comment]
@minecast from first glance that should work but you have some indent errors.

**Comment by minecast** [comment]
@13steinj thanks, and i did fix the indent errors and rebase again, here's what it looks like to me: http://i.imgur.com/rGxFD8r.png

**Comment by Pokechu22** [comment]
@minecast You've got tabs in the code.  Python doesn't like tabs (especially mixed with spaces) - that's going to cause things to freak out.  You'll need to replace the tabs here with spaces.

**Comment by minecast** [comment]
@Pokechu22 Thanks, fixed.

**Comment by Pokechu22** [comment]
@minecast You've still got some in jsontemplates.py, although the ones in link.py are fixed.

**Comment by minecast** [comment]
@Pokechu22 Good catch, thanks. Also fixed.

**Comment by minecast** [comment]
Since verdict can have more states than 'mod-removed' or 'admin-removed', was it a bad idea to use else?

**Comment by minecast** [comment]
@umbrae thoughts on current state of this pr?

**Comment by umbrae** [comment]
@minecast I like it. We're going through open source pulls next week, hopefully I'll be able to dive deeper then. I think this looks good though at first blush.

**Comment by umbrae** [comment]
:eyeglasses: @ajacksified @dforsyth This will change the response body for removed links in the API. Any qualms on that?

@minecast We'll probably need to take this PR internally and run with it, primarily because we'll need to "feature flag" this and test it with beta or employees to check if it breaks any API clients/RES/etc. If you want to be super awesome, you could check out how feature flags works: https://github.com/reddit/reddit/blob/master/r2/r2/config/feature/README.md 

Otherwise we can take this in house and tackle it. Thanks for the effort!

**Comment by ajacksified** [comment]
@umbrae no problem here, we'll just need to update our optimistic "set-text-to-[deleted]" code to the new proper "deleted" text.

**Comment by bsimpson63** [comment]
Sorry, but this repository is no longer accepting pull requests. https://www.reddit.com/r/changelog/comments/6xfyfg/an_update_on_the_state_of_the_redditreddit_and/

**Pull request closed without merging**